ml/squeezeseg/cli.py

```diff
@@ -0,0 +1,17 @@
+import click
+
+from ml.base import MLConfig
+from ml.squeezeseg.io import KittiSqueezeSegIO
+
+
+
+@click.command()
+@click.option('--config',
+              required=True,
+              help='The config file path.') 
+def npy2tfrecord(config):
+    config = MLConfig.from_file(config)
+    
+    # TODO(jxwulittlebean): The command line flags should override config here.
```

Comment:
in fact jiaxing's code can be added as a decorator to override config with flags